# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xf86-video-xgixp/hotfix.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xf86-video-xgixp/src/xg47_accel.c.vanilla	2026-02-16 13:07:26.130828762 +0100
+++ xf86-video-xgixp/src/xg47_accel.c	2026-02-16 13:08:41.184614290 +0100
@@ -82,7 +82,7 @@
 
 
 /* inner functions */
-static Bool XG47InitCmdList();
+static Bool XG47InitCmdList(ScrnInfoPtr pScrn);
 static void ResetClip();
 static void SetClip(XGIPtr, CARD32 left, CARD32 top, CARD32 right, CARD32 bottom);
 
